sass/directives/_headers.scss

```diff
@@ -129,34 +104,36 @@
     p {
       max-width: 28em;
       margin-bottom: 0;
-      font-size: $hero-text-size;
+      font-size: strip-units($hero-text-size) * 1rem;
```

Comment:
Yea the only reason why I decided to convert it from EM to REM was for some reason one of the headers had EM calculated at a much higher value. 

My guess is one of the root element's inside that header is different from the rest of the site. Maybe I can switch this back to  `$hero-text-size` and remove whatever styles are causing the issue. What do you think?